impress-org/givewp-next-gen
# Refactor: Reorganize Form Header controls

## Description

This PR reorders the Header, Heading and Description toggle controls related to the Visual Form Builders Header component. The Reorder was requested to be updated per design and also allows these controls to determine whether or not to show the related TextControls.

In addition, the Form Builder blocks with icons have had their corresponding svg elements replaced with the SVG & Path components provided by WordPress and the NextGen blocks have an updated fill color of ```currentColor``` so that the icons are visible when selected from the icon list. 

And lastly the Text block has had its title renamed to "Text Field"

## Affects

Design section Header controls & Block Icons.

## Visuals

https://github.com/impress-org/givewp-next-gen/assets/75056371/7299e427-0cd6-42f0-8cd5-b576f9acb9c6



TextField: 
<img width="1725" alt="Screen Shot 2023-07-25 at 7 34 18 PM" src="https://github.com/impress-org/givewp-next-gen/assets/75056371/1b4c346e-48de-4249-b716-62017312449f">

## Testing Instructions

- Click on a section block from the left menu to verify that it is visible when the selected background changes.
- Verify all icons still function properly.
- View new order of Header controls and test that the toggle controls show &/or hide the corresponding TextControls.
- Verify updated Text Field title.

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205189070743340
